src/pybind/mgr/volumes/module.py

```diff
@@ -334,6 +333,11 @@ class Module(orchestrator.OrchestratorClientMixin, MgrModule):
             'desc': "Fetch a export of a NFS cluster given the pseudo path/binding",
             'perm': 'r'
         },
+        {
+            'cmd': 'nfs export update ',
+            'desc': "Update an export of a NFS cluster by `-i <json_file>`",
+            'perm': 'rw'
+        },
```

Comment:
fyi, this relates to #38935 by @tchaikov

Reply:
@sebastian-philipp thanks for pointing this out. probably we can refactor this later on.